examples/howto/model_builder.myst.md

```diff
@@ -0,0 +1,292 @@
+* `model_type` : Model type tells us what kind of model it is. This in this case it outputs **Linear Model** 
+* `version` : In case you want to improve on models, you can keep track of model by their version. As the version changes the unique hash in the `id` also changes.
+* `sample_conifg` : It stores values of the sampler configuration set by user for this particular model.
+* `model_config` : It stores values of the model configuration set by user for this particular model.
+
+```{code-cell} ipython3
+idata = model.fit()
+```
+
+## Saving model to file
+
++++
+
+After fitting the model, we can probably save it to share the model as a file so one can use it again.
+To `save()` or `load()`, we can quickly call methods for respective tasks with the following syntax.
+
+```{code-cell} ipython3
+!rm linear_model_v1.nc
+```
+
+```{code-cell} ipython3
+:tags: []
+
+fname = "linear_model_v1.nc"
+model.save(fname)
+```
+
+This saves a file at the given path, and the name <br>
+A NetCDF `.nc` file that stores the inference data of the model.
+
++++
+
+## Loading a model
+
++++
+
+Now if we wanted to deploy this model, or just have other people use it to predict data, they need two things:
+1. the `LinearModel` class (probably in a .py file)
+2. the linear_model_v1.nc file
+
+With these, you can easily load a fitted model in a different environment (e.g. production):
+
+```{code-cell} ipython3
+model_2 = LinearModel.load(fname)
+```
+
+Note that `load()` is a class-method, we do not need to instantiate the `LinearModel` object.
+
+```{code-cell} ipython3
+model_2
+```
+
+## Prediction
+
++++
+
+Next we might want to predict on new data. The `predict()` method allows users to do posterior prediction with the fitted model on new data.
+
+Our first task is to create data on which we need to predict.
+
+```{code-cell} ipython3
+x_pred = np.random.uniform(low=1, high=2, size=10)
+prediction_data = pd.DataFrame({"input": x_pred})
+```
+
+`ModelBuilder` provides two methods for prediction:
+1. point estimates (the mean) with `predict()`
+2. full posterior prediction (samples) with `predict_posterior()`
+
+```{code-cell} ipython3
+pred_mean = model_2.predict(prediction_data)
+# samples
+pred_samples = model_2.predict_posterior(prediction_data)
+```
+
+After using the `predict()`, we can plot our data and see graphically how satisfactory our `LinearModel` is.
+
+```{code-cell} ipython3
+fig, ax = plt.subplots(figsize=(7, 7))
+ax.plot(
+    x_pred,
+    pred_mean["y"],
+    "x",
+    label="predict",
+)
+ax.set(title="Posterior predictive regression lines", xlabel="x", ylabel="y")
+plt.legend(loc=0);
+```
+
+```{code-cell} ipython3
+%load_ext watermark
+%watermark -n -u -v -iv -w -p pymc_experimental
+```
+
+## Authors
+* Authored by Shashank Kirtania and Thomas Wiecki in 2023.
```

Comment:
The page footer part is still missing: https://www.pymc.io/projects/docs/en/latest/contributing/jupyter_style.html#epilogue